__manifest__.py

```diff
@@ -14,6 +14,7 @@
         "views/openacademy_menu.xml",
         "views/openacademy_course_views.xml",
         "views/openacademy_session_views.xml",
+        "report/openacademy_session_report.xml",
         "views/res_partner_views.xml",
```

Comment:
I think it is a good practice to maintain an order according to the origin of the file type, this helps to locate files a little faster.

Ej.
- archivos_security
- archivos_data
- archivos_report
- archivos_wizard
- archivos_views

Already in each section you can order by models.